CharityDB/1 tables.sql

```diff
@@ -7,11 +7,15 @@ create table dbo.Donor(
     DonorId int not null identity primary key,
     FirstName varchar(25) not null constraint ck_Donor_FirstName_cannot_be_blank check(FirstName <> ''),
     LastName varchar(25) not null constraint ck_Donor_LastName_cannot_be_blank check(LastName <> ''),
-    Address varchar(50) not null constraint ck_Donor_Address_cannot_be_blank check(Address <> ''),
+    Address varchar(50) not null 
+        constraint ck_Donor_Address_cannot_be_blank check(Address <> '')
+        constraint u_Donor_Address_unique unique,
     PostalCode varchar(10) not null constraint ck_Donor_PostalCode_cannot_be_blank check(PostalCode <> ''),
     DonationAmount decimal(6,2) not null constraint ck_Donor_DonationAmount_must_be_between_1_and_5000 check(DonationAmount between 1 and 5000),
     Year int not null constraint ck_Donor_Year_must_be_between_2018_and_the_current_year check(Year between 2018 and year(getdate())),
     Season varchar(6) not null constraint ck_Donor_Season_must_be_autumn_or_spring check(Season in('Autumn', 'Spring')),
-    GiftAid bit not null
+    GiftAid bit not null,
+    constraint u_Donor_FirstName_LastName unique(FirstName, LastName),
```

Comment:
The name alone shouldn't be unique, join it to the add and postcode. per season.

Reply:
see comment above!
I'm confusing myself!!